scripts/src/changelogEntry.ts

```diff
@@ -1,8 +1,10 @@
-import nullthrows from 'nullthrows';
+import nullthrowsModule from 'nullthrows';
 
-import { CATEGORY_HEADERS, EntryCategory } from './changelog/consts';
-import { readAndParseChangelogAsync, writeChangelogAsync } from './changelog/file';
-import * as markdown from './markdown';
+import { CATEGORY_HEADERS, EntryCategory } from './changelog/consts.js';
+import { readAndParseChangelogAsync, writeChangelogAsync } from './changelog/file.js';
+import * as markdown from './markdown.js';
+
+const nullthrows = nullthrowsModule.default;
```

Comment:
if w need a workaround like this for such a simple package I would prefer to remove it entirely and just add and utll with the same name